bin/selenium-standalone

```diff
@@ -27,17 +28,33 @@ which('java', function javaFound(err, javaPath) {
   // everything after `selenium-standalone install [options] --` will be in argv._
   var seleniumArgs = argv._.slice(1);
 
-  // build a new map removing `_` from argv
+  // build a new map removing `_` and --config from argv
   var options = Object.keys(argv).reduce(function(prev, cur) {
-    if (cur !== '_') {
+    if ((cur !== '_') && (cur !== 'config')) {
       prev[cur] = argv[cur];
     }
 
     return prev;
   }, {});
 
-  // Merge in default options
-  options = merge({}, defaultConfig, options);
+  // If a config file was specified, load it
+  var configFromFile = {};
+  if (argv.config) {
+    try {
+      configFromFile = require(
+        path.isAbsolute(argv.config) ? argv.config : path.resolve(process.cwd(), argv.config)
+      );
+      if (typeof configFromFile !== 'object') {
+        throw new Error('Config file does not exports an object');
+      }
+    } catch(err) {
+      console.log('Error parsing config file :', err);
+      return;
+    }
+  }
+
+  // Merge default options, command line options and options from config file
+  options = merge({}, defaultConfig, options, configFromFile);
```

Comment:
`options` and `configFromFile` seem backwards to me. I would expect that CLI args have precedence over everything else.